app/default/data.py

```diff
@@ -0,0 +1,77 @@
+import json
+import os
+
+import requests
+from app.config import APPLICATION_STORE_API_HOST
+from app.config import FLASK_ROOT
+
+# Application Store Endpoints
+APPLICATION_ENDPOINT = "/applications/{application_id}"
+
+
+def get_data(endpoint: str):
+    if endpoint[:8] == "https://":
+        response = requests.get(endpoint)
+        if response.status_code == 200:
+            data = response.json()
+        else:
+            return None
```

Comment:
🤔 🔧 Maybe this should be its own encapsulated function - I would expect to have get_local_data, get_remote_data, and finally a get_data_handler function.

Reply:
good shout Jack- these helpers are also in the assessor repo so could do with refactoring there too 👍